roles/openshift_hosted/meta/main.yml

```diff
@@ -12,5 +12,24 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
-- openshift_common
-- openshift_hosted_facts
+- role: openshift_cli
+- role: openshift_hosted_facts
+- role: openshift_projects
+  openshift_projects: "{{ openshift_additional_projects | default({}) | oo_merge_dicts(hosted_projects | default({})) }}"
```

Comment:
Ran into problems accessing `hosted_projects` here from `roles/openshift_hosted/vars/main.yml` with ansible-1.9.4. `hosted_projects` likely needs to be defined here similar to how we do `os_firewall_allow` with other roles.